src/app/content/highlights/components/Confirmation.tsx

```diff
@@ -46,8 +48,13 @@ interface Props {
 
 // tslint:disable-next-line:variable-name
 const Confirmation = ({message, confirmMessage, confirmLink, always, onCancel, onConfirm, ...props}: Props) => {
+  const element = React.useRef<HTMLElement>(null);
+
+  useOnEsc(element, true, onCancel);
 
   return <Overlay
+    ref={element}
+    focus={true}
```

Comment:
this overlay is not being focused when it is added, i'm not sure this prop does anythign